src/components/dataset-details-page/index.tsx

```diff
@@ -143,12 +145,25 @@ const DatasetDetailsPage: FC<Props> = ({
       listAdministrativeUnits(spatialUris);
     }
 
+    let dataServiceParams = {};
     if (dataset?.uri) {
+      dataServiceParams = { dataseturi: dataset.uri };
       getDatasetsRelations({ referencesSource: dataset.uri });
-      getDataServicesRelations({ dataseturi: dataset.uri });
       getPublicServicesRelations({ isDescribedAt: dataset.uri });
     }
 
+    const accessUris =
+      dataset?.distribution?.flatMap(({ accessService }) =>
+        accessService?.map(({ uri }) => uri)?.filter(Boolean)
+      ) ?? [];
+    if (accessUris.length > 0) {
+      dataServiceParams = { ...dataServiceParams, uris: accessUris };
```

Comment:
Ikke sikker på om jeg skjønner helt hva feature går ut på. Du lager en filtrering på dataseturi og uris, fulltext-search filtrerer med AND. Jeg er ikke helt sikker på om denne filtreringen fungerer.

Reply:
Ah, ok. Så jeg må gjøre to søk etter datatjenester? En for servesDataset og en for uri?

Reply:
b959113